pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "Keystores":
```

Comment:
Should we have "Keystores" in uppercase? If so I think we should have it uppercase in all file.

I personally dont think this should be in uppercase.

Reply:
"keystores" will have a name you want them to have. Doesn't make sense for me to be all uppercase. I can substitute for `keystores`

Reply:
IMHO It is better that way.

Reply:
:+1: